provisioning/datastore.yml

```diff
@@ -21,7 +21,12 @@
   become: true
 
 - name: set default group write ACLs on /srv/datastore
-  command: /usr/bin/setfacl -m d:g::rwx /srv/datastore
```

Comment:
I removed the default ACL entries using `setfacl --remove-default /srv/publicdb/` in my VM, then ran the new provisioning and checked the ACL with `getfacl /srv/publicdb/`, the default ACL entries were correctly recreated using this module.